### ISSUE TITLE
Failed to execute 'play' on 'HTMLMediaElement': API can only be initiated by a user gesture on Chrome for Android

### ISSUE THREAD

**Issue by xcuben** [videojs/video.js#3131]
## Description
Hi Guys,
When I run Videojs player with videojs-vast-vpaid plugin on chrome for Android, ads are not playing because the page wait a gesture from the User.

### Error output
Failed to execute 'play' on 'HTMLMediaElement': API can only be initiated by a user gesture.

#### videojs
Videojs 5
#### browsers
 Chrome/48.0.2564.95
#### OSes
(Linux; Android 4.2.2; Lenovo B6000-F Build/JDQ39
### plugins
Videojs-vast-vpaid

**Comment by gkatsev** [comment]
Does it happen with any specific versions of videojs 5?
Also, are  you able to provide a reduced test case for this? Thanks.

**Comment by xcuben** [comment]
Hi gkatsev
I even tried to test it without the videojs-vast-vpaid Plugin 
Here is the page i am actually testing http://necib.team.netzathleten-media.de/23-02-2016/
I am currently using Videojs 5.0.0
Thanks

**Comment by gkatsev** [comment]
Can you try updating to a newer version of videojs 5.0? There have been a lot of fixes since 5.0 RC2.
Also, unfortunately, I do not currently have a physical 4.2 device to test with, though, newer versions of videojs seem to work in my emulator plus it works on newer android devices.

**Comment by xcuben** [comment]
Ok i will try it and let you know 
Thanks

**Comment by derek-knox** [comment]
@xcuben Did using a newer version alleviate the issue?

**Comment by varemenos** [comment]
This issues resurfaced for me as well. I think this was fixed* by a Chrome update.

*fixed for their use-case, broken for ours.

**Comment by derek-knox** [comment]
Just a heads up for future reference. We updated to 5.18.4, but a `$timeout` (AngularJS) was the root issue. I learned that the direct handler of a valid user gesture doesn't have to call `.play()` but may leverage an eventing approach to delegate the actual `.play()` call. In this case however, a `$timeout/setTimeout` ends up invalidating the user gesture. So if you have a custom UI that's responsible for playing audio/video, you must ensure this invalidation doesn't occur.
